policies/release-policy.md

```diff
@@ -0,0 +1,120 @@
+finalised release backlog and a release plan, both of which require sign-off by
+the OpenSSL Management Committee (OMC). Following the sign-off, the release
+plans are communicated externally.
+
+### 3. Development
+The Development phase centres on the implementation and execution of tasks
+identified in the release backlog. The phase can span from 20 to 24 weeks. To
+ensure timely release, commitments are periodically verified, and necessary
+adjustments are made. While the release backlog is generally maintained as
+initially defined, modification may be authorised by the release steering
+committee[^2] under specific circumstances, such as:
+* A task, initially underestimated, poses a risk of not being completed.
+* The emergence of an unexpected high-priority task necessitates immediate
+  resolution.
+
+Any modifications to the release backlog should be conducted with an intent to
+preserve the balance. Thus, whenever a task is added, an equivalent effort lower
+priority task(s) must be removed. Conversely, when a task is removed, another
+may be added to the release backlog provided that such a task fits within the
+available time and is not prioritised higher than any existing task within the
+release backlog.
+
+This phase concludes with an alpha release readiness check by the steering
+committee. Upon successful completion of this check, an alpha release is
+created, marking the beginning of the Alpha phase.
+
+### 4. Release
+The Release phase of OpenSSL library commences with the creation of an alpha
+release at the end of the Development phase and spans over a period of six
+weeks. In the Release phase, no new features can be added, nor can any code
+refactoring be planned to be undertaken. Effective communication at this stage
+is crucial to ensure the quality of the release and, consequently, the success.
+
+  * **Alpha**:
+  During this phase, feature development is complete and feedback from the
+  community is being actively addressed. Testing, bug fixing, and documentation
+  updates are ongoing. Any critical issues discovered may lead to the removal of
+  the offending code.
+  * **Beta**:
+  This phase signifies the library is nearing its final release. Testing remains
+  intensive, but the focus is on ensuring stability. While all identified
+  regressions and critical concerns have been addressed, only crucial,
+  last-minute fixes are expected.
+  * **Final**:
+  The Final phase is focused on release preparation. The tasks involved include
+  updating the list of known issues, tagging the code git tree, and issuing
+  digital signatures among others. The phase concludes with the new release
+  becoming publicly available and communicated.
+
+See [Release Requirements Policy] for details on each type of release.
+
+### 5. Support
+  * **Full Support**:
+  Upon the final release a one-year Full Support period is initiated for regular
+  releases, and a four-year period for LTS releases. During this phase, bugs and
```

Comment:
I'd spell out that it's a Full Support period for LTS releases.

Reply:
Okay. Make sense. Added.